flashinfer/cute_dsl/attention/gqa_decode.py

```
@@ -0,0 +1,2425 @@
+        tL_consumer_nbar = nbar(7, softmax_threads + correction_threads)
+        sM_final_nbar = nbar(9, correction_threads + reduction_threads)
+        sL_final_nbar = nbar(10, correction_threads + reduction_threads)
+        sO_final_nbar = nbar(11, correction_threads + tma_threads)
+        sM_mutex_nbar = nbar(12, softmax_threads * softmax_warpgroups)
+
+        # named barrier stage helper
+        def with_phase(nbar_, phase):
+            return nbar(nbar_.barrier_id + phase, nbar_.num_threads)
+
+        # Alias thread cooperatives
+        thr_cg = lambda t: CooperativeGroup(Agent.Thread, t)
```

Ruff (0.15.13)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `thr_cg` as a `def`

(E731)